compiler/damlc/lib/DA/Cli/Damlc.hs

```diff
@@ -317,6 +327,27 @@ execCompile inputFile outputFile opts = withProjectRoot' (ProjectOpts Nothing (P
         createDirectoryIfMissing True $ takeDirectory outputFile
         B.writeFile outputFile $ Archive.encodeArchive bs
 
+execLint :: FilePath -> Options -> Command
+execLint inputFile opts =
+  withProjectRoot' (ProjectOpts Nothing (ProjectCheck "" False)) $ \relativize ->
+  do
+    loggerH <- getLogger opts "lint"
+    inputFile <- toNormalizedFilePath <$> relativize inputFile
+    opts <- (setHlintDataDir <=< mkOptions) opts
```

Comment:
```suggestion
    opts <- setHlintDataDir =<< mkOptions opts
```
might be a bit more idiomatic but I don’t feel strongly about this at all so your choice :wink:

Reply:
Noted. Thanks!